README.md

```diff
@@ -1,10 +1,20 @@
+
+
+<div align="center">
+
 # cargo-generate
 
-> cargo, make me a project
+  <img src="https://github.com/cargo-generate/cargo-generate/raw/52be603bab5329b0ba90a19cafd58973f8781fa7/resources/logo.png" width="256"></a>
+
 
 [![Build status](https://github.com/cargo-generate/cargo-generate/workflows/Build/badge.svg)](https://github.com/cargo-generate/cargo-generate/actions?query=workflow%3ABuild+branch%3Amain+)
 [![crates.io](https://img.shields.io/crates/v/cargo-generate.svg)](https://crates.io/crates/cargo-generate)
 [![dependency status](https://deps.rs/repo/github/cargo-generate/cargo-generate/status.svg)](https://deps.rs/repo/github/cargo-generate/cargo-generate)
+[![Chat on Matrix](https://img.shields.io/matrix/cargo-generate:matrix.org?server_fqdn=matrix.org)](https://matrix.to/#/#cargo-generate:matrix.org)
```

Comment:
@orhun did you had similar issues with the martix room for git-cliff? Right now, the badge shows up like this:

<img width="308" alt="image" src="https://user-images.githubusercontent.com/329682/152892616-a356f59a-5f01-4925-ad2a-1309a509a56e.png">

Even thou when going to the link of the matrix badge the matrix chat is accessible.

Reply:
Yes, this happens due to chat history not being world readable, I reckon. There is a setting for it:

![Screenshot 2022-02-08 at 17-37-07 Element  5  git-cliff](https://user-images.githubusercontent.com/24392180/153009020-9c10fd13-2993-4dbc-82b9-4f3c74b214d9.png)